state/store.go

```diff
@@ -191,12 +193,14 @@ func LoadValidators(db dbm.DB, height int64) (*types.ValidatorSet, error) {
 				),
 			)
 		}
+		valInfo2.ValidatorSet.IncrementAccum(int(height - valInfo.LastHeightChanged)) // mutate
```

Comment:
What happens if LoadValidators is called multiple times before the validator set changes? Won't we keep incorrectly changing the accum?

Reply:
You're right, the result of LoadValidators doesn't seem to be saved anywhere. Excellent! This looks fine to merge then.